src/core/ext/client_config/client_channel.c

```diff
@@ -602,9 +602,11 @@ static bool pick_subchannel(grpc_exec_ctx *exec_ctx, grpc_call_element *elem,
     int r;
     GRPC_LB_POLICY_REF(lb_policy, "pick_subchannel");
     gpr_mu_unlock(&chand->mu);
-    const grpc_lb_policy_pick_args inputs = {calld->pollent, initial_metadata,
-                                             initial_metadata_flags,
-                                             &calld->lb_token_mdelem};
+    // TODO(dgq): use deadline for service config instead of inf_future for the
```

Comment:
I'm not sure what you mean by "deadline for service config".  Let's just say something like "Make this deadline configurable somehow".